core/services/reaper_test.go

```diff
@@ -28,9 +28,10 @@ func TestStoreReaper_ReapSessions(t *testing.T) {
 		wantReap bool
 	}{
 		{"current", time.Now(), false},
-		{"expired", time.Now().Add(-store.Config.SessionTimeout()), false},
-		{"almost stale", time.Now().Add(-store.Config.ReaperExpiration()), false},
-		{"stale", time.Now().Add(-store.Config.ReaperExpiration()).Add(-store.Config.SessionTimeout()), true},
+		{"expired", time.Now().Add(-store.Config.SessionTimeout().Duration()), false},
```

Comment:
These tests look like they mirror the implementation and might not be testing anything. Mind fixing them up?

Reply:
I think these are testing the reaping functionality (i.e., that `DeleteStaleSessions` is doing the right thing.)